add a very basic test for the set of global manifests

smalls opened the pull request:
Our tests now focus on specific demo manifests. This test loads the
default set of globally available manifests and makes sure we can see
some suggestions (as a proxy for those manifests not containing any
fatal errors).

smalls committed:
add a very basic test for the set of global manifests
Our tests now focus on specific demo manifests. This test loads the
default set of globally available manifests and makes sure we can see
some suggestions (as a proxy for those manifests not containing any
fatal errors).

mariakleiner commented on dev/test/specs/starter-test.js:
```diff
@@ -380,4 +382,40 @@ describe('test Arcs demo flows', function() {
+    waitForStillness();
+    browser.waitUntil(
+      () => {
+        const allSuggestions = pierceShadows([
```
const allSuggestions = pierceShadows([
      'div[slotid="suggestions"]',
      'suggestion-element'
]);
if (!allSuggestions.value || 0 == allSuggestions.value) {
       return false;
}

These line have a duplicate in acceptSuggestion method.. maybe create a helper method?

mariakleiner commented on dev/test/specs/starter-test.js:
```diff
@@ -380,4 +382,40 @@ describe('test Arcs demo flows', function() {
+    // indication that there aren't any major issues with globally available
+    // manifests.
+
+    browser.close();
```
i don't think this needs to call browser.close() because that was for closing the new arc tab in the other test cases.

smalls replied:
It was opening a tab the same as the others.. but, having explicit closes was lame. I've added code to close at the start.

mariakleiner reviewed (approved):
maybe this new test case should come first - they all might not be completely independent now (the number of open tabs etc), but if that one fails, everything else fails too anyway.

smalls committed:
don't rely on tests to clean up after themselves
Close extra windows before starting a new test, so a failing test won't break
other tests down the line.
Also a small refactor to remove duplication of selectors.

smalls commented:
They're actually pretty independent.. the rest of the tests use the `solo=` parameter, so they may pass even if the global test is failing.

This global test may end up adding more churn to the system.. changes not to this git repo can cause failures here, which is no longer the case with `solo=` and `canonical.manifest` which together have isolated the other tests to changes in the arcs-cdn repo.
